examples/controlnet/train_controlnet.py

```diff
@@ -1015,7 +1022,7 @@ def load_model_hook(models, input_dir):
                 noisy_latents = noise_scheduler.add_noise(latents, noise, timesteps)
 
                 # Get the text embedding for conditioning
-                encoder_hidden_states = text_encoder(batch["input_ids"])[0]
+                encoder_hidden_states = text_encoder(batch["input_ids"], return_dict=False)[0]
```

Comment:
Nit: For ControlNet training on SD, it won't matter (same for SDXL) as we never train the text encoder during ControlNet training. But keeping it this way doesn't hurt things. So, I am okay with it.